js/google_ima.js

```diff
@@ -869,13 +869,18 @@ require("../html5-common/js/utils/utils.js");
        */
       var _tryStartAdsManager = privateMember(function()
       {
-        if (!this.capturedUserClick && this.videoControllerWrapper && this.requiresMutedAutoplay())
+        //We do not want to mute if we are using ad rules, are handling the initial ad request
+        //for ad rules, and found no prerolls.
+        if (!this.capturedUserClick && this.videoControllerWrapper && this.requiresMutedAutoplay() &&
+            !(_usingAdRules && !this.hasPreroll && this.currentAMCAdPod &&
+            this.currentAMCAdPod.adType === _amc.ADTYPE.UNKNOWN_AD_REQUEST))
```

Comment:
It might help with clarity if this condition was set to a variable (assuming there's a word that can describe this group of conditions). It's a bit hard to reason about what the `if` is checking.

Reply:
werd. I agree.

Reply:
Sounds good. I'll make this change.